### ISSUE TITLE
Update less version

### ISSUE THREAD

**Issue by idealist1508** [Pscx/Pscx#26]
Less is currently at 487 version but Pscx includes the very old.
Would be a great idea to upgrade less to a newer version?

[Here](https://www.guysalias.tk/misc/less/) is a new Version.

**Comment by rkeithhill** [comment]
Thanks for the pointer.

**Comment by favorini** [comment]
Yes, please. Current version has a bug where it doesn't show any content if you are on the last line of the PowerShell command window (conhost on Windows 10 v1909 x64, anyhow).
